node/setup.go

```diff
@@ -233,12 +233,12 @@ func logNodeStartupInfo(state sm.State, pubKey crypto.PubKey, logger, consensusL
 	}
 }
 
-func onlyValidatorIsUs(state sm.State, pubKey crypto.PubKey) bool {
+func onlyValidatorIsUs(state sm.State, myAddr crypto.Address) bool {
```

Comment:
We still don't cover the scenario with more than one validator, one of which with more than 2/3.

Reply:
I'm pretty convinced that `onlyValidatorIsUs` is no longer necessary after this PR. We'll probably get rid of it. Not doing it here out of caution.